app/Package.php

```diff
@@ -119,14 +119,26 @@ public function toSearchableArray()
         // Temporarily truncate to prevent algolia from throwing a size exceeded exception
         $packageAttributes['readme'] = substr($packageAttributes['readme'], 0, 500);
         $packageAttributes['instructions'] = substr($packageAttributes['instructions'], 0, 500);
+        $packageAttributes['created_at'] = $this->created_at->timestamp;
 
         Arr::forget($packageAttributes, $this->excludeFromSearchIndex);
 
         // Add tags so we can filter them
         // @todo Make sure this is updated when tags are updated
         $packageAttributes['_tags'] = $this->tags->pluck('slug')->toArray();
 
-        return $packageAttributes;
+        return [
+            'id' => (string) $packageAttributes['id'],
+            'name' => (string) ($packageAttributes['name'] ?? ''),
+            'url' => (string) ($packageAttributes['url'] ?? ''),
+            'instructions' => (string) ($packageAttributes['instructions'] ?? ''),
```

Comment:
I'm not sure if this is any better, but potentially worth considering 🤷🏻

```suggestion
            'instructions' => data_get($packageAttributes, 'instructions', ''),
```